bundles/org.openhab.binding.rfxcom/README.md

```diff
@@ -964,10 +966,31 @@ A Thermostat3 device.
 
 #### Channels
 
-| Name        | Channel Type                        | Item Type | Remarks  |
-|-------------|-------------------------------------|-----------|----------|
-| command     | [command](#channels)                | Switch    |          |
-| signalLevel | [system.signal-strength](#channels) | Number    |          |
+| Name              | Channel Type                        | Item Type     | Remarks  |
+|-------------------|-------------------------------------|---------------|----------|
+| command           | [command](#channels)                | Switch        |          |
+| command2nd        | [command](#channels)                | Switch        |          |
+| control\*         | [tempcontrol](#channels)            | Rollershutter |          |
+| commandString\*\* | [commandString](#channels)          | String        |          |
+| signalLevel       | [system.signal-strength](#channels) | Number        |          |
+
+\* `control` supports:
+
+* UP
```

Comment:
Also supports `ON` and `OFF`.

Reply:
Are you sure not as far as what I read in the source code

Reply:
It did when it was a dimmer but not now its a rollershutter

Reply:
I based my comment on this implementation:
https://github.com/openhab/openhab2-addons/blob/1184fac60c8d51cc62cda7655bef622df6f2fb57/bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComThermostat3Message.java#L179

Reply:
Fair, that piece of code is unreachable because of the framework.